src/main/java/org/apache/mesos/curator/CuratorStateStore.java

```diff
@@ -354,6 +355,35 @@ public void storeProperty(final String key, final byte[] value) throws StateStor
         }
     }
 
+    public void storePropertyAsObj(final String key, final Object obj) throws StateStoreException {
```

Comment:
I'll create an AbstractStateStore with these methods, so all implementations can share the functionality.